CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Updated the README file to remove outdated information.
+- Moved API client classes to `csm-api-client` library, and added that new
```

Comment:
Suggest clarifying a bit because not all were moved, just `APIGatewayClient`, `CFSClient`, `HSMClient`, `VCSRepo`, and associated error classes and the `SATSession` functionality was refactored mostly into `UserSession`. Perhaps just "Refactored to use common `APIGatewayClient`, `CFSClient`, `HSMClient`, `VCSClient`, and `UserSession` classes from `csm-api-client` library."